src/UnitTests/GitHub.App/ViewModels/PullRequestCreationViewModelTests.cs

```diff
@@ -33,4 +34,31 @@ public class PullRequestCreationViewModelTests : TempFileBaseClass
         var unused = ms.Received().CreatePullRequest(repository, vm.PRTitle, String.Empty, vm.SourceBranch, vm.TargetBranch);
     }
 
+    [Fact]
+    public void TemplateIsUsedIfPresent()
+    {
+        var serviceProvider = Substitutes.ServiceProvider;
+        var service = Substitute.For<IPullRequestService>();
```

Comment:
BTW, if you do this instead of `service.GetPullRequestService`, the instance that's returned by `service.GetService(typeof(IPullRequestService))` is going to be different than the instance you have here. It doesn't matter for this case since there's no codepath that ends up querying the service provider, but if there was, things would break, just fyi.